src/Admin/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -484,6 +490,11 @@ private void UpdateOrganization(Organization organization, OrganizationEditModel
             organization.GatewayCustomerId = model.GatewayCustomerId;
             organization.GatewaySubscriptionId = model.GatewaySubscriptionId;
         }
+
+        if (model.PlanType == PlanType.Free && organization.GatewaySubscriptionId is not null)
+        {
+            organization.GatewaySubscriptionId = null;
+        }
```

Comment:
Are we sure we want to do this? Clay's comment in Slack was:

> I would rather the code be resilient enough for the canceled subscriptionId to remain present just like it is for Premium memberships, which will also mitigate the need for us to go back and manually remove this from all of the existing downgraded Free accounts.

Reply:
The customerId is retain which contains details of all existing subscription in stripe. Retaining This will make the subscription details to have two subscription on the admin console, when the account is cancelled and downgraded to free. (The free and the cancelled plan)

Reply:
No problem, I'm less familiar with how these Ids are used, so I'll leave it with you to discuss with Clay 👍

Reply:
I have discussed with Clay and He insist of retaining the old subscription Id. I have remove the code